nx-dev/ui-conference/src/lib/conf-schedule-short.tsx

```diff
@@ -42,8 +42,7 @@ export function ConfScheduleShort(): JSX.Element {
       title: 'Keynote',
       description: '',
       speakers: ['Juri Strumpflohner', 'Victor Savkin'],
-      videoUrl:
-        'https://www.youtube.com/live/IQ5YyEYZw68?si=TQRfAG7CtNbd3Xu4&t=250',
+      videoUrl: 'https://youtu.be/a7ggRWYqX3s',
```

Comment:
This links to a video that doesn't exist any more

Reply:
fixed